packages/package.json

```diff
@@ -14,7 +14,7 @@
     "npm": "6.x"
   },
   "scripts": {
-    "prebuild": "rm -Rf dist-*",
+    "prebuild": "rm -Rf \"*/dist\"",
```

Comment:
`rm -rF ./*/dist` does not work ?

Reply:
Windows likes globs to have double quotes.

Reply:
Actually, trying it again now, it doesn't work with the quotes... Removing them now.